modules/cli/pages/cbstats/cbstats-config.adoc

```diff
@@ -114,11 +117,27 @@ ep_item_freq_decayer_chunk_duration:                   20
 ep_item_freq_decayer_percent:                          50
 ep_item_num_based_new_chk:                             true
 ep_keep_closed_chks:                                   false
+ep_magma_commit_point_every_batch:                     false
```

Comment:
Assuming this was run on a EE setup, could we add a note as CE users won't see "ep_magma_*"?